### PR TITLE
Add indentation to preferences

### PR DESCRIPTION
**Pull request by MathuraMG**
This PR 
* adds indentation to preferences
* allows indentation and font to be a input
* changes the return syntax in preferences reducer

### CODE REVIEW

**Comment by catarak** [comment]
screen shot 2016-07-11 at 5 56 25 pm

Some of the styles are a bit off:
* The box holding the numbers for the font size and tab indentation should have no border radius
* This plus and minus icons are off center (to fix this I gave the plus and minus icons their own line height but it seems like you took this out accidentally?)
* The font color for the inactive spaces/tabs is too dark, should be #B5B5B5
* Everything is a little tight, see Sketch designs for correct spacing

**Comment by MathuraMG** [comment]
I fixed the first four points, could you have a look and let me know if it's ok ? 
I was going to add the styling to remove the dotted border in the end as there are more preferences - would that be ok?

screen shot 2016-07-11 at 7 15 02 pm

Also, I realized I didn't add validation to ensure that the font size and indentation do not go below 0 - I will make that in the next PR.

File: client/styles/components/_preferences.scss [hunk 1]
```diff
@@ -35,7 +35,8 @@
 .preference {
 	display: flex;
 	flex-wrap: wrap;
-	justify-content: space-between;
+	padding-bottom: #{40 / $base-font-size}rem;
+	border-bottom: 2px dashed $light-button-border-color;
```
Comment (catarak) [inline client/styles/components/_preferences.scss]: To remove the border from the last preference, all you have to do is change this line to:
```scss
& + & {
  border-top: 2px dashed $light-button-border-color;
}
```

**Comment by MathuraMG** [comment]
My bad, I didn't understand the styling right - added the change you mentioned!

**Comment by MathuraMG** [comment]
fixing merge conflicts.

**Comment by catarak** [comment]
Btw, fixed some minor styling for this PR, see [this commit](https://github.com/catarak/p5.js-web-editor/commit/d96e7132dc5bd73807cb0b8e4700bda5540e144d).

**Pull request merged**